client/src/views/LoginRegisterContainer/Nav.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+/**
+ * @function Nav
+ * @memberof Nav.js
+ * @returns {void}
+ * Nav holds routes to login and register components
+ */
+function Nav() {
```

Comment:
1. es6 arrow style function
2. Nav is not proper variable name.... write it out fully `Navigation`